pkg/util/execdetails/execdetails.go

```diff
@@ -752,20 +752,42 @@ func (crs *CopRuntimeStats) RecordOneCopTask(address string, summary *tipb.Execu
 		storeType: crs.storeType,
 		BasicRuntimeStats: BasicRuntimeStats{
 			tiflashScanContext: TiFlashScanContext{
-				totalDmfileScannedPacks:             summary.GetTiflashScanContext().GetTotalDmfileScannedPacks(),
-				totalDmfileSkippedPacks:             summary.GetTiflashScanContext().GetTotalDmfileSkippedPacks(),
-				totalDmfileScannedRows:              summary.GetTiflashScanContext().GetTotalDmfileScannedRows(),
-				totalDmfileSkippedRows:              summary.GetTiflashScanContext().GetTotalDmfileSkippedRows(),
-				totalDmfileRoughSetIndexCheckTimeMs: summary.GetTiflashScanContext().GetTotalDmfileRoughSetIndexCheckTimeMs(),
-				totalDmfileReadTimeMs:               summary.GetTiflashScanContext().GetTotalDmfileReadTimeMs(),
-				totalCreateSnapshotTimeMs:           summary.GetTiflashScanContext().GetTotalCreateSnapshotTimeMs(),
-				totalLocalRegionNum:                 summary.GetTiflashScanContext().GetTotalLocalRegionNum(),
-				totalRemoteRegionNum:                summary.GetTiflashScanContext().GetTotalRemoteRegionNum(),
-				totalLearnerReadMs:                  summary.GetTiflashScanContext().GetTotalLearnerReadMs(),
-				totalDisaggReadCacheHitSize:         summary.GetTiflashScanContext().GetTotalDisaggReadCacheHitSize(),
-				totalDisaggReadCacheMissSize:        summary.GetTiflashScanContext().GetTotalDisaggReadCacheMissSize()}}, threads: int32(summary.GetConcurrency()),
+				totalDmfileDataScannedRows:          summary.GetTiflashScanContext().GetDmfileDataScannedRows(),
```

Comment:
How about removing these `total` as well?

However, for time metrics, please keep the `total` prefix in the name, because we don't know whether it is an avg, or a total, or a min, or a max, if it is not specified.